lib/generators/trln_argon/templates/local_env.yml

```diff
@@ -15,3 +15,5 @@ PAGING_LIMIT: '250'
 FACET_PAGING_LIMIT: '50'
 # Uncomment for Solr 9, where truncation is needed to get under maxClauseCount constraint
 # ENABLE_QUERY_TRUNCATION: 'true'
+# Uncomment to add route \logs?debug=true&lines=50 (lines parameter is optional)
```

Comment:
I'd remove the `debug=true` from this comment, since it's no longer used for this route.

Reply:
Just fixed this. The ci is still running.